src/documentation/install.page

```diff
@@ -8,32 +8,32 @@ sort_info: 2
 lang: en
 ---
 
-# Install Uniqush System
+# Installing Uniqush and its Dependencies
 
-## Install uniqush-push
+## Installing uniqush-push
 
-uniqush-push now is the only component in the Uniqush System.
+uniqush-push now is the only component in Uniqush.
 
 It is higly recommanded to install from **binary** package on **64-bit** system.
 
 ### Install Redis
 
-The only dependency of uniqush-push is [redis], which is a key-value database.
+The only dependency of uniqush-push is [Redis], which is a key-value database.
 
-- On debian-based system (including [debian], [ubuntu], [mint], etc.), run:
+- On Debian-based systems (including [Debian], [Ubuntu], [Mint], etc.), run:
     sudo apt-get install redis-server
-- On yum-based system (including [Redhat Enterprise Linux], [fedora], [centos], etc.), run:
+- On yum-based systems (including [Redhat Enterprise Linux], [Fedora], [CentOS], etc.), run:
     sudo yum install redis-server
-- For other systems, please visit [redis] website for more information
+- For other systems, please visit the [Redis] website for more information.
 
 ### Install uniqush-push
 
-- According to your system, download the corresponding uniqush-push binary package from [download page].
-- If you are using debian-based system, you can install .deb package: 
+- Download the latest uniqush-push binary package appropriate for your system from the [download page] or you can build it from source. Go 1.6+ is recommended.
+- If you are using a Debian-based system, you can install the DEB:
     > sudo dpkg -i uniqush-push\_VERSION\_ARCH.deb
-- Or, if you are using yum/rpm-based system, you can install .rpm package:
+- If you are using yum/rpm-based system, you can install RPM:
     > sudo rpm -ivh uniqush-push-VERSION-ARCH.rpm
-- Otherwise, you can always install uniqush-push with .tar.gz package:
+- Otherwise, you can always install uniqush-push using the tarball:
     * unpack the file:
         > tar zxvf uniqush-push-VERSION-ARCH.tar.gz
     * enter that directory and execute *install.sh* under that directory:
```

Comment:
instead of italics for program/file names, use monospace if possible.

Also, the block quotes aren't working as intended. http://uniqush.org/install.html

I think you use triple `~~~` above and below a block to render it as a monospace code block, assuming maruku is being used by the setup.

https://github.com/gettalong/webgen-website/blob/master/src/documentation/reference/extensions/content_processor/maruku.page

Or if block quotes are still intended, try adding an empty line between the two things and putting it at the same indentation of other text in the bullet?